frontend/src/components/Search.js

```diff
@@ -33,5 +33,6 @@ const Wrapper = styled.div`
   grid-template-rows: min-content min-content min-content min-content 1fr;
   Button {
     background-color: #008080;
+    border-radius: 0px;
```

Comment:
do you need to set the border-radius here?

Reply:
Yes. I have to overwrite the standard Border-radius from Material UI.